photon-ml/src/main/scala/com/linkedin/photon/ml/function/svm/SmoothedHingeLossLinearSVMFunction.scala

```diff
@@ -12,33 +12,19 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-package com.linkedin.photon.ml.function
+package com.linkedin.photon.ml.function.svm
```

Comment:
This class is a bit wiggy, as it's a function that we use with our GLM framework to approximate an SVM. What would you recommend?